compiler/ast/parser.nim

```diff
@@ -1264,22 +1286,27 @@ proc postExprBlocks(p: var Parser, x: ParsedNode): ParsedNode =
     p.getTok
     p.skipComment(result)
     if p.tok.tokType notin {tkOf, tkElif, tkElse, tkExcept}:
```

Comment:
Pre-existing, but I think `tkFinally` might be missing here?

Reply:
pretty sure it is, I'll add it in later or separately.

Reply:
pretty sure it is, I'll add it in later or separately.

Reply:
The `finally` is a must, a test will confirm it, but without that present it'll never do that same indent loop for finally at the bottom.

----

The "real" question is do we enforce only a single `finally` and it being last? I'm leaning strongly towards yes. Even though it's not for exception handling, we should ensure there is a certain amount of regularity to the code or we'll end up with people not trusting the syntax they're reading, also a matter of identity (that looks like nimskull code). Scala 2 had a lot of issues with this.

Don't have to finalize it now, but something to mull over. Conceptually, I consider this similar to operator precedence. Lastly, it's easier to alleviate a restriction later than add one.